aep/general/0124/aep.yaml

```
@@ -1,6 +1,6 @@
 ---
 id: 124
-state: reviewing
+state: approved
 slug: resource-association
 created: 2023-01-22
```

```suggestion
created: 2024-03-15
```

IIRC we update these, but @rofrankel @mkistler do you have thoughts?

---

I'm going to change for now. IMO, creating a stub shouldn't define the AEP's creation.